### DISCUSSION TITLE
ESModules url imports are only supported in the experimental ESModule preset.

### DISCUSSION THREAD

**Discussion by leo-petrucci** [codesandbox/codesandbox-client#6758] [Q&A]
I'm trying to `require` a file from a URL and I get this error back.

How do I access this experimental ESModule preset?

Example Codesandbox: https://rctzc6.csb.app/

**Comment by sebastien** [comment]
Would be interested in the answer as well, I can see the line in question in the client code https://github.com/codesandbox/codesandbox-client/blob/master/packages/sandpack-core/src/manager.ts#L777 but can't find where to define that preset.

**Reply by victorlongon** [reply]
I would also to like how to set that option. I have been digging the whole day with no avail.

**Comment by ggorlen** [comment]
See also https://github.com/codesandbox/codesandbox-client/discussions/6449

**Comment by R-W-C** [comment]
So you must be enable a preset which does not seems to be available. How curious.
